tile/targets/gpu/amd.jsonnet

```diff
@@ -31,24 +31,25 @@ local PARAMS = {
           // Define the stripe passes
           passes: [
             // First, we place all the initial buffer in global memory (DRAM)
-            {
-              name: 'loc_program',
-              pass: {
-                '@type': 'type.vertex.ai/vertexai.tile.codegen.proto.LocateMemoryPass',
-                reqs: ['program'],
-                loc: { devs: [{ name: 'GLOBAL', units: [{ offset: 0 }] }] },
-              },
-            },
-
-            {
-              name: 'loc_main',
-              pass: {
-                '@type': 'type.vertex.ai/vertexai.tile.codegen.proto.LocateMemoryPass',
-                reqs: ['main'],
-                loc: { devs: [{ name: 'GLOBAL', units: [{ offset: 0 }] }] },
-              },
-            },
-
+            
+            // {
```

Comment:
Let's remove these lines if we don't need them anymore (we can always recover them via git history)

Reply:
removed.